### DISCUSSION TITLE
Parameter identification function does not work.

### DISCUSSION THREAD

**Discussion by liussmzk** [Open-Systems-Pharmacology/Forum#1220] [Questions & Problems]
I am a new user of PK-SIM.  I am trying to optimize my model but the parameter identification function does not work. After I added the parameter and ran visual feedback function, a error message came out and asked me restart from parameter identification. A screenshot has been attached. Thank you. 

[issues.pptx](https://github.com/Open-Systems-Pharmacology/Forum/files/8819415/issues.pptx)

**Comment by msevestre** [comment]
@liussmzk 
Did you actually start the Parameter Identification?

![image](https://user-images.githubusercontent.com/1041237/171540202-b73be56f-2a7b-4a5c-a13b-220a5704c776.png)

Please, please, especially if you are new to the software, make sure you read the documentation
https://docs.open-systems-pharmacology.org/

specifically
https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#simple-example
